roles/tilemaker/templates/process-openmaptiles.lua

```diff
@@ -246,6 +246,28 @@ function way_function(way)
 		way:Attribute("class", h)
 		SetBrunnelAttributes(way)
 
+        -- Construction
```

Comment:
This file has pretty weird indentation but since we want to eventually upstream this, please also indent the same way: tabs.

Reply:
It looks perfectly okay in my editor this is why this always happens... Is GitHub checks the whitespaces a specific way?

Reply:
You are setting spaces but they rest uses tabs. You need to find the setting in your editor to visualise tabs and force your editor to use tabs instead. I know, this is _very_ annoying.

Reply:
Actually, i checked the new code and it looks good now!